extension/content/customPersonaEditor.js

```diff
@@ -263,6 +263,9 @@ var CustomPersonaEditor = {
       case PersonaService.FIREFOX_ID:
  	window.close();
         break;
+      case PersonaService.SEAMONKEY_ID:
+    window.close();
```

Comment:
Ok, then please indent it at least correctly. line 267 having less indent than 266 just makes no sense.